commands.md

```diff
@@ -32,6 +32,7 @@
 | [podman-pod(1)](/docs/podman-pod.1.md)                   | Simple management tool for groups of containers, called pods              ||
 | [podman-pod-create(1)](/docs/podman-pod-create.1.md)     | Create a new pod                                                          ||
 | [podman-pod-ps(1)](/docs/podman-pod-ps.1.md)             | List the pods on the system                                               ||
+| [podman-restart](/docs/podman-pod-restart.1.md)          | Restarts one or more pods                                                 ||
```

Comment:
probably should  be "Restart one or more pods", at least to be consistent with remove, start, stop, etc.